src/components/QuestionPlay/WithoutOptions.tsx

```diff
@@ -18,14 +20,50 @@ export default function WithoutOptions({
   options,
   isAttempted,
   setIsAnswerRevealed,
-  setSelectedChoices,
+  submitResponse,
+  continueGame,
 }: Props) {
-  function handleSelectedChoice(choice: number[]) {
-    setSelectedChoices(choice);
+  const [isUndoTimerRunning, setIsUndoTimerRunning] = useState(false);
+  const [time, setTime] = useState(0);
+  const [isUndoClicked, setIsUndoClicked] = useState(false);
+  const timerRef = useRef(null as any);
+  const userResponse = useRef(null as any);
 
-    setTimeout(() => {
-      document.getElementById('btnSubmitResponse')?.click();
-    }, 100);
+  useEffect(() => {
+    if (isUndoTimerRunning) {
+      timerRef.current = setTimeout(() => {
+        if (time === 4) {
+          handleContinueClick(userResponse.current);
```

Comment:
Why do we click continue automatically? We should just hide the undo button here and stop timer. Let user click continue as they might have not finished seeing the result.

Reply:
fixed